contrib/forms/xmlformgen/xslt/report.php.xslt

```diff
@@ -107,7 +107,7 @@ $lists = array(]]></xsl:text>
               $value = $dateparts[0];
             }
 
-	    echo "<td><span class='bold'>";
+	    echo "<td style='width:24%'><span class='bold'>";
```

Comment:
Another thing to mention is that the current code does following via the pdf libraries:
1. PDF documents -> native pdf gets appending to the pdf report
2. non-pdf documents -> Outputted to html that gets converted to pdf and then appended to the pedf report